platform/pulp/platform/logs.py

```diff
@@ -0,0 +1,165 @@
+"""
```

Comment:
I left the other version of this file in server/. I'm interested to see whether others think we should try to keep the Pulp 2 setup working with a Pulp 3 checkout or not. If not (which is what I'm in favor of), I'll just move the server/ version here.